### ISSUE TITLE
100 Helpful Python Tips

### ISSUE THREAD

**Issue by rhiriyappa** [rhiriyappa/data-science-articles#15]
# TLDR;

#Article Link
https://towardsdatascience.com/100-helpful-python-tips-you-can-learn-before-finishing-your-morning-coffee-eb9c39e68958

[100 Helpful Python Tips You Can Learn Before Finishing Your Morning Coffee | by Fatos Morina | May, .pdf](https://github.com/rhiriyappa/data-science-articles/files/6490201/100.Helpful.Python.Tips.You.Can.Learn.Before.Finishing.Your.Morning.Coffee.by.Fatos.Morina.May.pdf)

#Key Takeaway
Python is quite popular nowadays, mainly due to its simplicity, and easiness to learn.
You can use it for a wide range of tasks like data science and machine learning, web development, scripting, automation, etc.
Since this is a pretty long article and you want to reach its end before you finish your coffee, why don’t we just start?
